HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTrackstersCLUE3DHigh_cfi.py

```diff
@@ -115,19 +103,31 @@
     pluginPatternRecognitionByFastJet = cms.PSet(
         algo_verbosity = cms.int32(0),
         antikt_radius = cms.double(0.09),
-        eid_input_name = cms.string('input'),
-        eid_min_cluster_energy = cms.double(1),
-        eid_n_clusters = cms.int32(10),
-        eid_n_layers = cms.int32(50),
-        eid_output_name_energy = cms.string('output/regressed_energy'),
-        eid_output_name_id = cms.string('output/id_probabilities'),
         minNumLayerCluster = cms.int32(5),
         type = cms.string('FastJet')
     ),
+
+    pluginInferenceAlgoTracksterInferenceByDNN = cms.PSet(
+        algo_verbosity = cms.int32(0),
+        onnxPIDModelPath = cms.FileInPath('RecoHGCal/TICL/data/RecoHGCal-TICL/ticlv4/onnx_models/energy_id_v0.onnx'),
+        onnxEnergyModelPath = cms.FileInPath('RecoHGCal/TICL/data/RecoHGCal-TICL/ticlv4/onnx_models/energy_id_v0.onnx'),
+        eid_min_cluster_energy = cms.double(1),
+        eid_n_layers = cms.int32(50),
+        eid_n_clusters = cms.int32(10),
+        doPID = cms.int32(0),
+        doRegression = cms.int32(0),
+        type = cms.string('TracksterInferenceByDNN')
+    ),
+    pluginInferenceAlgoTracksterInferenceByANN = cms.PSet(
+        algo_verbosity = cms.int32(0),
+        type = cms.string('TracksterInferenceByANN')
+
```

Comment:
For completeness, this PSet is missing the entries coming from
https://github.com/cms-sw/cmssw/blob/20b37d420e3d466539e652874237466089a05043/RecoHGCal/TICL/plugins/TracksterInferenceByDNN.cc#L140-L142

 and the associated modifier for ticl_v5